calendar/cmd/grpc-server/main.go

```diff
@@ -0,0 +1,5 @@
+package main
```

Comment:
what if we are restricted by the number of processes in the system? This folder is more for the infrastructure layer rather than the application one.

Reply:
ye, it is more infrastructure one. what would you suggest here?

Reply:
agreed to use a single executable (process) to serve 2 servers: HTTP & gRPC. Both ways are acceptable, with this one, developers have more clarity and responsibility for a solution.